Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -28,9 +28,9 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
 
     _backedTextInputView = [[RCTUITextView alloc] initWithFrame:self.bounds];
     _backedTextInputView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-#if TARGET_OS_OSX // TODO(macOS GH#774)
+#if TARGET_OS_OSX // [macOS
     self.hideVerticalScrollIndicator = NO;
-    _scrollView = [[RCTUIScrollView alloc] initWithFrame:self.bounds]; // TODO(macOS ISS#3536887)
+    _scrollView = [[RCTUIScrollView alloc] initWithFrame:self.bounds]; // [macOS]
```

Comment:
Is this tag needed if its part of another macOS tag block

Reply:
I guess not.